internal_transfers/actions/tsconfig.json

```diff
@@ -1,18 +1,14 @@
 {
-    "compileOnSave": true,
-    "compilerOptions": {
-        "module": "commonjs",
-        "noImplicitReturns": true,
-        "noUnusedLocals": true,
-        "outDir": "out",
-        "sourceMap": true,
-        "strict": true,
-        "target": "es2020"
-    },
-    "exclude": [
-        "**/*.spec.ts"
-    ],
-    "include": [
-        "**/*"
-    ]
-}
\ No newline at end of file
+  "compileOnSave": true,
+  "compilerOptions": {
```

Comment:
You might want to also add;
`"noImplicitAny": true,`
This would force you to use the type system, as otherwise there are many cases where type system implicitly would assume "whatever you put here goes". It tends to cascade quite fast.